NoteFind/Alerts/ErrorAlerts.h

```diff
@@ -0,0 +1,33 @@
+//
+//  ErrorAlerts.h
+//  NoteFind
+//
+//  Created by Khloe Wright on 7/22/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "LoginViewController.h"
+#import "FeedViewController.h"
+#import "RegisterViewController.h"
+#import "NewNoteViewController.h"
+#import <UIKit/UIKit.h>
+
+@class LoginViewController;
+@class FeedViewController;
+@class RegisterViewController;
+@class NewNoteViewController;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface ErrorAlerts : NSObject
++ (void)loginFailure:(LoginViewController *)loginVC;
```

Comment:
You probably don't need a separate method for each alert type, I think you could get away with a single `showAlertForErrorType:(ErrorAlertType)errorType` method and then define an `NS_ENUM` to list out all the different error options https://nshipster.com/ns_enum-ns_options/#ns_enum